src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -422,6 +435,35 @@ export function fetchRequestDetails(id) {
   };
 }
 
+export function fetchConfirmedAppointmentDetails(id) {
+  return async (dispatch, getState) => {
+    const state = getState();
+    const { appointmentDetails } = state.appointments;
+    const futureAppointments = selectFutureAppointments(state);
+    const appointment =
+      appointmentDetails[id] ||
+      futureAppointments?.find(appt => appt.id === id);
+
+    dispatch({
+      type: FETCH_REQUEST_DETAILS,
```

Comment:
Should these two dispatches in here use the new action names you created?

Reply:
Fixed.